argoproj/base/argo-cd.yaml

```diff
@@ -3,6 +3,9 @@ kind: Application
 metadata:
   name: argo-cd
   namespace: argocd
+  ### Just as a prevention, we do not want ArgoCD to self delete
+  # finalizers:
+  #   - resources-finalizer.argocd.argoproj.io
```

Comment:
Meant to still be commented out?

Reply:
I though that leaving the config commented out with the explanation on top was I good idea, this way people that look at the apps aren't asking themselves why is there no finalizer to this app or why isn't the sync enabled? was it on purpose or someone forgot?

Since it is a demo setup, I think it is interesting to explicitly explain why it is commented.

ditto for all commented sections in Apps.

What do you think?